lib/options.ts

```diff
@@ -28,7 +28,7 @@ export interface MillifyOptions {
  * Default options for Millify.
  */
 export const defaultOptions: MillifyOptions = {
-  decimalSeparator: ".",
+  decimalSeparator: null,
```

Comment:
After using `Intl.NumberFormat`, I find the `decimalSeparator` property to be somewhat outdated and cumbersome. Therefore, I propose we update this option to accept locales instead:

```js
locales?: string | string[]
```

This would enable us to use the locale parameter directly in `NumberFormat`. It would require a breaking change, but I'm fine with pushing a new major version.